File: invesdwin-context-parent/invesdwin-context-javascript/src/test/java/de/invesdwin/context/javascript/tests/InputsAndResultsTestShort.js
print("getShort")
if(typeof getShort !== 'undefined')
	throw "getShort already defined!"
getShort = putShort
getShortType = typeof getShort
print(getShortType)
print(getShort)
if(getShortType !== 'number')
	throw "getShort not number!"

print("getShortVector")
if(typeof getShortVector !== 'undefined')
	throw "getShortVector already defined!"
getShortVector = putShortVector
getShortVectorType = typeof getShortVector[0]
print(getShortVectorType)
print(getShortVector)
if(getShortVectorType !== 'number')
	throw "getShortVector not number!"

print("getShortVectorAsList")
if(typeof getShortVectorAsList !== 'undefined')
	throw "getShortVectorAsList already defined!"
getShortVectorAsList = putShortVectorAsList
getShortVectorAsListType = typeof getShortVectorAsList[0]
print(getShortVectorAsListType)
print(getShortVectorAsList)
if(getShortVectorAsListType !== 'number')
	throw "getShortVectorAsList not number!"

print("getShortMatrix")
if(typeof getShortMatrix !== 'undefined')
	throw "getShortMatrix already defined!"
getShortMatrix = putShortMatrix
getShortMatrixType = typeof getShortMatrix[0][0]
print(getShortMatrixType)
print(getShortMatrix)
if(getShortMatrixType !== 'number')
	throw "getShortMatrix not number!"

print("getShortMatrixAsList")
if(typeof getShortMatrixAsList !== 'undefined')
	throw "getShortMatrixAsList already defined!"
getShortMatrixAsList = putShortMatrixAsList
getShortMatrixAsListType = typeof getShortMatrixAsList[0][0]
print(getShortMatrixAsListType)
print(getShortMatrixAsList)
if(getShortMatrixAsListType !== 'number')
	throw "getShortMatrixAsList not number!"

File: invesdwin-context-parent/invesdwin-context-scala/src/test/java/de/invesdwin/context/scala/tests/InputsAndResultsTestShort.scala
System.out.println("getShort")
if(getShort != null)
	throw new Exception("getShort already defined!")
val getShort = putShort
val getShortType = getShort.getClass()
System.out.println(getShortType)
System.out.println(getShort)
if(getShortType != classOf[java.lang.Short])
	throw new Exception("getShort not Short!")

System.out.println("getShortVector")
if(getShortVector != null)
	throw new Exception("getShortVector already defined!")
val getShortVector = putShortVector.asInstanceOf[Array[Short]]
val getShortVectorType = getShortVector(0).getClass()
System.out.println(getShortVectorType)
System.out.println(getShortVector)
if(getShortVectorType != classOf[Short])
	throw new Exception("getShortVector not Short!")

System.out.println("getShortVectorAsList")
if(getShortVectorAsList != null)
	throw new Exception("getShortVectorAsList already defined!")
val getShortVectorAsList = putShortVectorAsList.asInstanceOf[Array[Short]]
val getShortVectorAsListType = getShortVectorAsList(0).getClass()
System.out.println(getShortVectorAsListType)
System.out.println(getShortVectorAsList)
if(getShortVectorAsListType != classOf[Short])
	throw new Exception("getShortVectorAsList not Short!")

System.out.println("getShortMatrix")
if(getShortMatrix != null)
	throw new Exception("getShortMatrix already defined!")
val getShortMatrix = putShortMatrix.asInstanceOf[Array[Array[Short]]]
val getShortMatrixType = getShortMatrix(0)(0).getClass()
System.out.println(getShortMatrixType)
System.out.println(getShortMatrix)
if(getShortMatrixType != classOf[Short])
	throw new Exception("getShortMatrix not Short!")

System.out.println("getShortMatrixAsList")
if(getShortMatrixAsList != null)
	throw new Exception("getShortMatrixAsList already defined!")
val getShortMatrixAsList = putShortMatrixAsList.asInstanceOf[Array[Array[Short]]]
val getShortMatrixAsListType = getShortMatrixAsList(0)(0).getClass()
System.out.println(getShortMatrixAsListType)
System.out.println(getShortMatrixAsList)
if(getShortMatrixAsListType != classOf[Short])
	throw new Exception("getShortMatrixAsList not Short!")
